packages/auth/src/Login/ForceEnrollMfa.tsx

```diff
@@ -112,7 +110,7 @@ export const ForceEnrollMfa: FC<ForceEnrollMfaProps> = (props) => {
               <FButton
                 type='submit'
                 size='large'
-                fullWidth={false}
+                fullWidth={true}
```

Comment:
why making the button full width?

Reply:
@AntonReact

Reply:
<img width="569" alt="Screen Shot 2021-01-15 at 9 49 37 AM" src="https://user-images.githubusercontent.com/54889005/104696505-0bdf9b00-5717-11eb-9097-27e92f713903.png">
I've added "back to login" button everywhere. Have changed full width to make everything looks the same

Reply:
@davidantoon 
There are several screens in the jira task. You can check out if the design is fine.
https://frontegg.atlassian.net/browse/FR-1170